bravado/fido_client.py

```diff
@@ -226,7 +226,7 @@ def prepare_request_for_twisted(request_params):
                 'url': string,
                 'timeout': float,  # optional
                 'connect_timeout': float,  # optional
-                'allow_redirects': bool,  # optional
+                'follow_redirects': bool,  # optional
```

Comment:
I think this was incorrectly added in my PR per the lack of other changes in this file. I couldn't get this to work, but I don't believe I later removed it from the documentation.